<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDIA_NEUANLAGE</class>
 <widget class="QDialog" name="CDIA_NEUANLAGE">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Neuanlage</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="lineEdit_bez"/>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="lineEdit_seriennr"/>
   </item>
   <item row="4" column="0">
    <widget class="QComboBox" name="comboBox_abt"/>
   </item>
   <item row="6" column="0">
    <widget class="QComboBox" name="comboBox_gruppe"/>
   </item>
   <item row="8" column="0">
    <widget class="QComboBox" name="comboBox_standort"/>
   </item>
   <item row="10" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_ansch"/>
   </item>
   <item row="12" column="0">
    <widget class="QDoubleSpinBox" name="doubleSpinBox_wertak"/>
   </item>
   <item row="14" column="0">
    <widget class="QDateEdit" name="dateEdit_anscham"/>
   </item>
   <item row="16" column="0">
    <widget class="QLineEdit" name="lineEdit_notiz"/>
   </item>
   <item row="18" column="0">
    <widget class="QComboBox" name="comboBox_zustand"/>
   </item>
   <item row="20" column="0">
    <widget class="QComboBox" name="comboBox_verantwortlicher">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Bezeichnung</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Seriennummer</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Abteilung</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Gruppe</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Standort</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Wert Anschaffung</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Wert Aktuell</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Angeschafft am</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Notiz</string>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Zustand</string>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>verantwortlicher</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDIA_NEUANLAGE</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDIA_NEUANLAGE</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
